deployments/common/ap-policy-definition.yaml

```diff
@@ -0,0 +1,874 @@
+                  type: object
+                headers:
+                  items:
+                    properties:
+                      base64Decoding:
+                        type: boolean
+                      checkSignatures:
+                        type: boolean
+                      htmlNormalization:
+                        type: boolean
+                      id:
+                        type: string
+                      kind:
+                        type: string
+                      lastUpdateMicros:
+                        type: integer
+                      mandatory:
+                        type: boolean
+                      maskValueInLogs:
+                        type: boolean
+                      name:
+                        type: string
+                      normalizationViolations:
+                        type: boolean
+                      percentDecoding:
+                        type: boolean
+                      selfLink:
+                        type: string
+                      type:
+                        type: string
+                      urlNormalization:
+                        type: boolean
+                    type: object
+                  type: array
+                json-profiles:
+                  items:
+                    properties:
+                      defenseAttributes:
+                        properties:
+                          maximumArrayLength:
+                            pattern: any|\d+
+                            type: string
+                          maximumStructureDepth:
+                            pattern: any|\d+
+                            type: string
+                          maximumTotalLengthOfJSONData:
+                            pattern: any|\d+
+                            type: string
+                          maximumValueLength:
+                            pattern: any|\d+
+                            type: string
+                          tolerateJSONParsingWarnings:
+                            type: boolean
+                        type: object
+                      description:
+                        type: string
+                      hasValidationFiles:
+                        type: boolean
+                      name:
+                        enum:
+                        - Default
+                        type: string
+                    type: object
+                  type: array
+                json-validation-files:
+                  items:
+                    properties:
+                      contents:
+                        type: string
+                      fileName:
+                        type: string
+                      isBase64:
+                        type: boolean
+                    type: object
+                  type: array
+                methods:
+                  items:
+                    properties:
+                      name:
+                        enum:
```

Comment:
Custom methods can be added that are not defined in the RFC. They should not be limited to an enum list.